app/controllers/discover.js

```diff
@@ -233,13 +201,9 @@ export default ApplicationController.extend({
     }),
 
     actions: {
+
         addFilter(type, filterValue) {
-            let filters = this.get('facetFilters');
-            let currentFilter = this.get(type) ? this.get(type) : [];
-            let value = currentFilter.indexOf(filterValue) > -1 ? [] : [filterValue];
-            let filter = termsFilter(type, Array.prototype.concat(value, currentFilter));
-            filters.set(type, filter);
-            this.send('filtersChanged', filters);
+            console.log(type, filterValue);
```

Comment:
Well that's what will have to be used when adding filters from the search result... (I can't figure out a good way to do that currently).